plugins/00_dokku-standard/subcommands/version

```diff
@@ -4,7 +4,7 @@ source "$PLUGIN_CORE_AVAILABLE_PATH/common/functions"
 source "$PLUGIN_AVAILABLE_PATH/checks/functions"
 source "$PLUGIN_AVAILABLE_PATH/config/functions"
 source "$PLUGIN_AVAILABLE_PATH/nginx-vhosts/functions"
-source "$PLUGIN_CORE_AVAILABLE_PATH/proxy/functions"
+source "$PLUGIN_AVAILABLE_PATH/proxy/functions"
```

Comment:
This plugin isn't a proxy implementation. It's the interface plugin to call the configured proxy plugin.